README.md

```diff
@@ -34,6 +34,11 @@ let app = new EmberApp(defaults, {
     }
 ```
 
+From `ember-cli-uniq@0.3` to `ember-cli-uniq@0.4`:
+
+* Replace bower import of `uniq` to npm import of `uniplaces-uniq`
```

Comment:
I would suggest something more in the likes of:

Importing `uniq` using bower is now deprecated. Import `uniplaces-uniq` using npm with the following command:
```bash
$ npm install uniplaces-uniq
```